reaper/util.py

```diff
@@ -134,7 +134,7 @@ def create_archive(content, arcname, metadata=None, outdir=None):
         if metadata is not None:
             zf.comment = json.dumps(metadata, default=metadata_encoder)
         for fn, fp in files:
-            zf.write(fp, os.path.join(arcname, fn))
+            zf.write(fp, os.path.join(arcname, fn) if rootdir else fn)
```

Comment:
Thanks for catching that. We'll update the viewer, which was recently merged, btw, to match our standard convention of one directory level (https://github.com/flywheel-io/frontend/issues/1093).

Let's rip this out after the demo.